server/app/views/admin/programs/ProgramIndexViewV2.java

```diff
@@ -351,17 +351,13 @@ private static Instant extractLastUpdated(
     return program.lastModifiedTime().orElse(Instant.EPOCH);
   }
 
-  Tag renderViewLink(ProgramDefinition program) {
-    // TODO(#1238): Rather than navigating to the program link
-    // consider renaming the action and copying the link to the clipboard
-    // or opening it in a new tab.
-    String viewLink =
+  Tag renderCopyProgramLink(ProgramDefinition program) {
+    String programLink =
         baseUrl
             + controllers.applicant.routes.RedirectController.programByName(program.slug()).url();
-    ContainerTag button =
-        makeSvgTextButton("View", Icons.VISIBILITY_SVG_PATH)
-            .withClass(AdminStyles.TERTIARY_BUTTON_STYLES);
-    return asRedirectButton(button, viewLink);
+    return makeSvgTextButton("Copy link", Icons.CONTENT_COPY_SVG_PATH)
```

Comment:
suggestion (blocking): maybe `Copy shareable link`?

Reply:
We want program admins to understand:

1. the link is a deep link, and this the only place to get it (copying the URL after visiting a specific program links to a specific version)
1. the link is human-readable (i.e. `/programs/utility-discount-program/` instead of `/programs/12/blocks/1`) which is what all users see when actually visiting a program with their browser

Reply:
Spoke with @rahaghas , who suggested "Share link". I additionally added some hover text that is a riff on the text we originally had within the original version of the UI.

If it's acceptable to y'all, I'd suggest going with this, getting feedback, and iterating.